pwchem/protocols/VirtualDrugScreening/protocol_ligand_characterization.py

```diff
@@ -0,0 +1,144 @@
+# **************************************************************************
+# *
+# * Authors: ...
```

Comment:
Write here your name (if you want) as the author and also mine and my email in case anyone needs help. 
Daniel Del Hoyo Gomez (ddelhoyo@cnb.csic.es)